esp-hal/src/usb_serial_jtag.rs

```diff
@@ -792,9 +778,8 @@ impl UsbSerialJtagTx<'_, Async> {
     }
 }
 
+#[instability::unstable]
```

Comment:
We shouldn't use `#[unstable]` on inherent impl blocks, the attribute should be placed on each function. But `read_bytes_async` is also private so I'm not sure why we have the attribute here at all.